lib/screens/v2/candidate_generator/elevator.ex

```diff
@@ -27,10 +28,15 @@ defmodule Screens.V2.CandidateGenerator.Elevator do
   def candidate_instances(
         config,
         now \\ DateTime.utc_now(),
-        elevator_closure_instances_fn \\ &ElevatorClosures.elevator_status_instances/1
+        elevator_closure_instances_fn \\ &ElevatorClosures.elevator_status_instances/1,
+        evergreen_content_instances_fn \\ &Evergreen.evergreen_content_instances/2
       ) do
-    [header_instance(config, now), footer_instance(config)] ++
-      elevator_closure_instances_fn.(config)
+    Enum.concat([
+      header_instance(config, now),
+      footer_instance(config),
```

Comment:
I'm confused as to how this works. `header_` and `footer_instance` return a single widget struct, which is not an enumerable, and `Enum.concat/1` expects an enum _of_ enums. Doing the equivalent of this call in IEx results in an error:

```
iex(1)> Enum.concat([:a, :b, [:c, :d], [:e, :f]])
** (Protocol.UndefinedError) protocol Enumerable not implemented for :a of type Atom
    (elixir 1.15.7) lib/enum.ex:1: Enumerable.impl_for!/1
    (elixir 1.15.7) lib/enum.ex:166: Enumerable.reduce/3
    (elixir 1.15.7) lib/enum.ex:4387: anonymous fn/3 in Enum.concat_enum/1
    (elixir 1.15.7) lib/enum.ex:2510: Enum."-concat_enum/1-lists^foldl/2-1-"/3
    (elixir 1.15.7) lib/enum.ex:2510: Enum.concat_enum/1
    iex:1: (file)
```

Reply:
[\*cough\*](https://mbtace.sentry.io/issues/6092494293)

Reply:
🫥